data/XML/vocations.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <vocations>
 	<vocation id="0" clientid="0" name="None" description="none" gaincap="10" gainhp="5" gainmana="5" gainhpticks="6" gainhpamount="1" gainmanaticks="6" gainmanaamount="1" manamultiplier="4.0" attackspeed="2000" basespeed="220" soulmax="100" gainsoulticks="120" fromvoc="0">
```

Comment:
Since we're considering just one new attribute in this PR, I think it's best to add it to this ugly-looking giant `vocation` element as `allowPvp="0/1"`.

I don't think we'll find a pattern that is consistent for different files because there's too much history. Inventing a new "flags" thing at this point seems unnecessary to me.

Reply:
> Since we're considering just one new attribute in this PR, I think it's best to add it to this ugly-looking giant `vocation` element as `allowPvp="0/1"`.
> 
> I don't think we'll find a pattern that is consistent for different files because there's too much history. Inventing a new "flags" thing at this point seems unnecessary to me.

but if I'm not mistaken he intends to add new flags in new PR, related to bank restriction (and drop loot?)

Reply:
Maybe. Still, I see no difference between them and the existing attributes. Should we move all existing ones into "flags"? Of course not. So what's the point of adding "flags". Also, I think flags are mostly boolean because it's a reference to... flags, that are usually raised (or not). The only real flags in TFS were [the ones stored as bitmask](https://my-aac.org/flags/). We're not doing that here and hopefully we never do that.